remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/filter/AbstractJPAFilterConverterFactory.java

```diff
@@ -0,0 +1,54 @@
+package org.infinispan.query.remote.impl.filter;
+
+import org.infinispan.commons.CacheException;
+import org.infinispan.protostream.ProtobufUtil;
+import org.infinispan.protostream.SerializationContext;
+import org.infinispan.protostream.config.Configuration;
+import org.infinispan.query.remote.client.BaseProtoStreamMarshaller;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author gustavonalle
+ * @since 8.1
+ */
+public class AbstractJPAFilterConverterFactory {
```

Comment:
Not much value in making this a base class for those factories. It's probably better if we make it a package local utility class. JPACacheEventFilterConverterFactory should use it too.